pilot/pkg/proxy/envoy/discovery.go

```diff
@@ -184,6 +191,9 @@ type DiscoveryService struct {
 	sdsCache *discoveryCache
 
 	RestContainer *restful.Container
+
+	// true if a full push is needed after debounce. False if only EDS is required.
+	fullPush bool
```

Comment:
needIncrementalPush

Reply:
Keep in mind we'll have incremental CDS, LDS, RDS - so this is just an intermediary step. 
If you look at the rest of the PR - full is used pretty much everywhere to indicate that a full recomputation of the config is needed. That's I don't want to change individual structs or fields - I won't be able to follow the PR.